src/theme/card.css.ts

```diff
@@ -43,7 +41,22 @@ export const cardTitleImage = style({
 // recipe for the card style
 // this ensures we have themeable variations for the card
 export const cardStyle = recipe({
-  base: card,
+  base,
+  variants: {
+    size: {
+      default: {
+        maxWidth: pxToRem(336),
+        minWidth: pxToRem(336),
+      },
+      wide: {
+        maxWidth: pxToRem(1200),
+        minWidth: pxToRem(1200),
+      },
```

Comment:
on mobile this will break

Reply:
i don't think you need min width

Reply:
https://github.com/ory/elements/pull/126/files#diff-f9b41df7e9b5477726dcc81876e220a391da277699d25ef6e4b7ebcb14f7cb2aL22-L31

No, there is a media query in the base styles. But yes, min-width is an issue, I'll remove it.